tests/replication/test_pusher_shard.py

```diff
@@ -92,8 +96,8 @@ def test_send_push_single_worker(self):
         )
 
         self.make_worker_hs(
-            "synapse.app.pusher",
-            {"start_pushers": False},
+            "synapse.app.generic_worker",
+            {"pusher_instances": ["synapse.app.generic_worker"]},
```

Comment:
This doesn't seem correct / doesn't match the shared config above. What's with that?

Reply:
Copy paste weirdness. Fixed up now.